pkg/features/features.go

```diff
@@ -19,8 +19,6 @@ package features
 import utilfeature "k8s.io/apiserver/pkg/util/feature"
 
 const (
-	// owner: @verult
```

Comment:
I think we should keep this so we know who are the feature owners like in k/k

Reply:
@msau42 dont we record it in `OWNERS` file instead of individual source file? I thought thats the general practice.

Reply:
It's not a file-by-file ownership as much as a feature ownership that can span several files/packages

Reply:
@davidz627 not sure I got your last comment . Are you saying invidual file mention  of owner is fair enough ?

Reply:
What I'm saying is
`OWNERS` file -> specifies ownership over some file or folders (related files)
`features.go` comments -> specifies ownership over some feature that may span several unrelated files, and does not imply ownership over the files in general